frontend/src/components/GoalForm/TopicsInfoLIst.scss

```diff
@@ -0,0 +1,8 @@
+.smart-hub-topics-info-header {
+  font-size: 16px;
```

Comment:
if you incorporate the two suggestions I made in your template you can delete this file

Reply:
Thanks!

Reply:
no - thank YOU